scripts/pi-hole/js/settings-teleporter.js

```diff
@@ -48,7 +48,9 @@ function importZIP() {
         if (data.error.hint !== null) $("#modal-import-error-message").text(data.error.hint);
       } else if ("files" in data) {
         $("#modal-import-success").show();
-        $("#modal-import-success-title").text("Import successful");
+        $("#modal-import-success-title").text(
+          "Import successful, please run pihole -g to update gravity."
```

Comment:
Maybe a hyperlink to the web interface's own gravity update page?

Reply:
Just throwing ideas around:

Use a button or move the text+link from the alert title

<details>
<summary>View examples</summary>

![image](https://github.com/pi-hole/web/assets/1385443/4d3a7bf8-b279-45ff-a022-69e9dc064d95)

![image](https://github.com/pi-hole/web/assets/1385443/57d9733a-05f3-42b6-9e29-586a6a89ff59)

</details>

Reply:
I like the idea to move the text to the bottom. The button alone does not indicate that users should run gravity.

Reply:
How about...

![Screenshot from 2023-11-03 18-50-46](https://github.com/pi-hole/web/assets/16748619/433a51fe-d82d-4cf7-ad32-be4e4ec0d011)

Reply:
I think it's good.